lib/improver/spotdata/site_data.py

```diff
@@ -103,7 +103,7 @@ def from_file(self, file_path):
         """
         if file_path.endswith('.json'):
             data_file = open(file_path, 'r')
-            site_data = json.load(data_file)
+            site_data = json.load(data_file)["sites"]
```

Comment:
Are we expecting something else to be in the site_data file, other than a list of sites?

If you remove:
```
{
	"sites": 
```
from the start of the json, and:
```
}
```
from the end, the json still validates using https://jsonlint.com/

Reply:
I've removed the usage of a dictionary within the json file, so that a list is used instead.